### DISCUSSION TITLE
[Q&A] Force an Inventory

### DISCUSSION THREAD

**Discussion by YeyeX64** [glpi-project/glpi-agent#566] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hello on GLPI-Agent 1.5, i can Force an Inventory drirectly through browser (see sreenshot below)

![Capture d’écran du 2023-12-22 14-32-56](https://github.com/glpi-project/glpi-agent/assets/86892253/1c70f17d-434a-49b2-9b4f-d486d72fbe07)

but i the new version 1.7 version (1.6 too) i can't.

![Capture d’écran du 2023-12-22 14-33-09](https://github.com/glpi-project/glpi-agent/assets/86892253/eb457541-e028-45f0-bf94-0c5543eb69ca)

Did I miss something ?

**Comment by g-bougard** [comment, marked as answer]
Hi @YeyeX64 

when speaking about a glpi-agent problem, don't forget to tell on which OS the agent is installed.

Here I can only guess looking at the exposed `1.7-1` version this is a linux agent.

Linux agents by default won't trust localhost, so you must configure it in `httpd-trust` configuration parameter if you're trying to access the httpd interface from the localhost.

If you're accessing from another computer, you must check you added the related computer or its related ip range in the `httpd-trust` configuration.

So check what is configured i, `httpd-trust` in configuration files, probably in the `/etc/glpi-agent/conf.d` folder, but this depends on the way you installed the agent.

**Comment by YeyeX64** [comment]
Oui c’est bien sous linux (debian)
Thanks, super merci  !!  ça fonctionne en spécifiant ip ou range.
